Src/WindowsServer/WindowsServer.Shared/Implementation/DataContracts/AzureInstanceComputeMetadata.cs

```diff
@@ -0,0 +1,177 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.Implementation.DataContracts
+{
+    using System;
+    using System.Globalization;
+    using System.Linq;
+    using System.Runtime.Serialization;
+    using System.Text.RegularExpressions;
+
+    /// <summary>
+    /// Class representing the returned structure from an Azure Instance Metadata request
+    /// for Compute information.
+    /// </summary>
+    [DataContract]
+    internal class AzureInstanceComputeMetadata
+    {
+        private static int resourceGroupNameLengthMax = 90;
```

Comment:
Can these (int and string fields) be `const` instead of `static`?

Reply:
Yes, and they've been changed.